src/components/CreateMenu.js

```diff
@@ -84,5 +114,6 @@ class CreateMenu extends PureComponent {
 }
 
 CreateMenu.propTypes = propTypes;
+CreateMenu.defaultProps = defaultProps;
 CreateMenu.displayName = 'CreateMenu';
```

Comment:
I know class components don't need `displayName`, but I'm not sure about PureComponents. Can the display name be removed?

Reply:
Do you know?